karma.conf.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
```

Comment:
> It is just simpler to outsource all this work to something else. Like Karma.

To me it looks more complex, not simpler, but maybe I'm missing something. Can you show me how it simplifies config with some before and after examples?